bindata/v4.1.0/profiles/LifecycleAndUtilization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "PodLifeTime":
+     enabled: true
+     params:
+       podLifeTime:
+         maxPodLifeTimeSeconds: 86400 #24 hours
+  "RemovePodsHavingTooManyRestarts":
+     enabled: true
+     params:
+       podsHavingTooManyRestarts:
+         podRestartThreshold: 100
+         includingInitContainers: true
+  "LowNodeUtilization":
+     enabled: true
+     params:
+       nodeResourceUtilizationThresholds:
+         thresholds:
+           "cpu" : 20
+           "memory": 20
+           "pods": 20
+         targetThresholds:
+           "cpu" : 50
```

Comment:
Why 50, why not higher?

Reply:
that's just the number picked from the upstream examples, we could go higher like `60` but I think going much above that would really reduce what nodes are considered overutilized

Reply:
Let's start with 50, and we can always tweak those. It's not that what we'll pick here is final.